app/views/shelters/show.html.erb

```diff
@@ -11,11 +11,13 @@
   <section class="review-section">
     <%= link_to ("New Review"), "/shelters/#{@shelter.id}/reviews/new" %>
     <% @shelter.reviews.each do |review| %>
-      <%= image_tag ( review.review_image ) %>
-      <%= review.title %>
-      Rating: <%= review.rating %>
-      <%= review.content %>
+      <div id="review-<%= review.id %>">
```

Comment:
always trying to remember where and when to interpolate a certain way... this gets me a lot